tests/unit/Endpoints/Webhooks/LocationTest.php

```diff
@@ -0,0 +1,333 @@
+            );
+
+            if (null === $scope) {
+                $this->assertTrue($isLocationUpdated);
+                $this->assertFalse($isAddressUpdated);
+                $this->assertFalse($areOpeningHoursUpdated);
+                $this->assertFalse($areOpeningSpecialHoursUpdated);
+            } else if (LocationUpdateImport::ADDRESS_SCOPE === $scope) {
+                $this->assertFalse($isLocationUpdated);
+                $this->assertTrue($isAddressUpdated);
+                $this->assertFalse($areOpeningHoursUpdated);
+                $this->assertFalse($areOpeningSpecialHoursUpdated);
+            } else if (LocationUpdateImport::OPENING_HOUR_SCOPE === $scope) {
+                $this->assertFalse($isLocationUpdated);
+                $this->assertFalse($isAddressUpdated);
+                $this->assertTrue($areOpeningHoursUpdated);
+                $this->assertFalse($areOpeningSpecialHoursUpdated);
+            } else if (LocationUpdateImport::OPENING_SPECIAL_HOUR_SCOPE === $scope) {
+                $this->assertFalse($isLocationUpdated);
+                $this->assertFalse($isAddressUpdated);
+                $this->assertFalse($areOpeningHoursUpdated);
+                $this->assertTrue($areOpeningSpecialHoursUpdated);
+            }
+
+            $location = $updatedLocation;
+            $address = $updatedAddress;
+            $openingHours = $updatedOpeningHours;
+            $openingSpecialHours = $updatedOpeningSpecialHours;
+
+            unset(
+                $updatedLocation,
+                $scope,
+                $updatedLocation,
+                $updatedAddress,
+                $updatedOpeningHours,
+                $updatedOpeningSpecialHours,
+                $isLocationUpdated,
+                $isAddressUpdated,
+                $areOpeningHoursUpdated,
+                $areOpeningSpecialHoursUpdated
+            );
+        }
+    }
+
+    /**
+     * Testing location status (activate/deactivate) task
+     */
+    public function testLocationStatus()
+    {
+        $this->initApiConnection();
+
+        $storeKeeperId = $this->activateLocation();
+
+        $location = LocationModel::getByStoreKeeperId($storeKeeperId);
+
+        $this->assertIsArray($location);
+        $this->assertTrue((bool) $location['is_active']);
+
+        $this->deactivateLocation();
+
+        $location = LocationModel::getByStoreKeeperId($storeKeeperId);
+
+        $this->assertFalse((bool) $location['is_active']);
+
+        $this->activateLocation();
+
+        $location = LocationModel::getByStoreKeeperId($storeKeeperId);
+
+        $this->assertTrue((bool) $location['is_active']);
+    }
+
+    /**
+     * Testing location deactivate task
+     */
+    public function testLocationDeactivated()
+    {
+        $this->initApiConnection();
+
+        $storeKeeperId = $this->deactivateLocation();
+
+        $location = LocationModel::getByStoreKeeperId($storeKeeperId);
+
+        $this->assertNull($location);
+    }
```

Comment:
I think this does not test anything since the location was never activated. Maybe better to remove this since it's covered in `testLocationStatus` anyway, or in this test you can activate first before deactivating